style.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+printf "[ISORT: sheetstorm]\n"
+find-files-to-refactor.sh | xargs isort -sl
+printf "\n"
+
+printf "[BLACK: sheetstorm]\n"
+find-files-to-refactor.sh | xargs black --line-length 120
```

Comment:
Why just finding them instead of instantly fixing them? :)

Reply:
@rwrzesien You mean find them all first, and then without `xargs` command fix them all? :)

Reply:
Or just apply `black` to all files in repository?